components/mpas-ocean/src/mode_forward/mpas_ocn_time_integration_si.F

```diff
@@ -270,7 +271,11 @@ subroutine ocn_time_integrator_si(domain, dt)!{{{
          highFreqThicknessNew,        &! high frq thick at new     time
          lowFreqDivergenceCur,        &! low frq div    at current time
          lowFreqDivergenceNew,        &! low frq div    at new     time
-         tracerGroupIdealAgeMask       ! mask for resetting surface ideal age
+         tracerGroupIdealAgeMask,     &! mask for resetting surface ideal age
+         vertVelocityNonhydroNew,         &! Vertical Velocity at new time
+         vertVelocityNonhydroCur,         &! Vertical Velocity at current time
+         nonhydrostaticPressure,      &! nhpressure at new time
+         nonhydrostaticPressureOld     ! nh pressure at current time
```

Comment:
same comment as above, should probably rename the nonhydrostaticPressure arrays for consistency with other fields.